com.unity.mobile.notifications/Editor/NotificationSettingsManager.cs

```diff
@@ -0,0 +1,342 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.CompilerServices;
+using UnityEditor;
+using UnityEngine;
+
+using Unity.Notifications.iOS;
+
+[assembly: InternalsVisibleTo("Unity.Notifications.Tests")]
+namespace Unity.Notifications
+{
+    [HelpURL("Packages/com.unity.mobile.notifications/documentation.html")]
+    internal class NotificationSettingsManager : ScriptableObject
+    {
+        internal const string ASSET_PATH = "Editor/com.unity.mobile.notifications/NotificationSettings.asset";
```

Comment:
Code style - kAssetPath, or k_AssetPath, also since it's used only in this class, should be private